frontend/src/features/settings/preferences/VideosPollUserSettingsForm.tsx

```diff
@@ -43,6 +48,16 @@ const VideosPollUserSettingsForm = () => {
     DEFAULT_RATE_LATER_AUTO_REMOVAL
   );
 
+  // Recommendations (page)
+  const [recoDefaultUnsafe, setRecoDefaultUnsafe] = useState(false);
```

Comment:
When reloading the page, after having set my configuration to True, I see it showing false for a second, then showing my selected value. I'd recommend not displaying until the settings are fetched.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1553/commits/dab7cc70653f759fe23a1ad83c727faf74efbb68